### PR TITLE
Integration with RADICAL-Pilot 

### PR DESCRIPTION
**Pull request by cspgdds**

### CODE REVIEW

File: fabsim/base/fab.py [hunk 1]
```diff
@@ -1257,70 +1263,168 @@ def run_ensemble(
...
+
+    job(dict(ensemble_mode=False, label="radical-PJ-header", NoEnvScript=True))
+    env.in_ensemble_mode = False
+    env.batch_header = backup_header
+    env.NoEnvScript = False
+            
+def run_qcg(job_scripts_to_submit: list, venv: bool):
+    print(
+        Panel.fit(
+            "NOW, we submitting PJ jobs",
```
Comment (djgroen) [inline fabsim/base/fab.py:1368]: Seems to be a typo on this line (I think it may be in one other spot).

**Comment by arabnejad** [comment]
Hi @cspgdds :grin:
just got email for this MR,
one comment (I just had a quick look), `lower()` method returns the lowercase string from the given string
so, the `PJ_TYPE.lower()` variable should be compared with lowercase string

`env.PJ_TYPE.lower() == "QCG"` should be `env.PJ_TYPE.lower() == "qcg"`

**Comment by arabnejad** [comment]
we already load all templates with [`local_templates_path`](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L88)

why we need to load `radical-PJ-py` with [radical_PJ_py](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L96) and `radical-PJ-header`with [radical_PJ_header](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L97)  variables separately  :confused:

**Comment by cspgdds** [comment]
> we already load all templates with [`local_templates_path`](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L88)
> 
> why we need to load `radical-PJ-py` with [radical_PJ_py](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L96) and `radical-PJ-header`with [radical_PJ_header](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L97) variables separately 😕

**Comment by cspgdds** [comment]
Accidentally closed it

**Comment by mzrghorbani** [comment]
> Hi @cspgdds 😁 just got email for this MR, one comment (I just had a quick look), `lower()` method returns the lowercase string from the given string so, the `PJ_TYPE.lower()` variable should be compared with lowercase string
> 
> `env.PJ_TYPE.lower() == "QCG"` should be `env.PJ_TYPE.lower() == "qcg"`



> Hi @cspgdds 😁 just got email for this MR, one comment (I just had a quick look), `lower()` method returns the lowercase string from the given string so, the `PJ_TYPE.lower()` variable should be compared with lowercase string
> 
> `env.PJ_TYPE.lower() == "QCG"` should be `env.PJ_TYPE.lower() == "qcg"`

**Comment by djgroen** [comment]
> we already load all templates with [`local_templates_path`](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L88)
> 
> why we need to load `radical-PJ-py` with [radical_PJ_py](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L96) and `radical-PJ-header`with [radical_PJ_header](https://github.com/djgroen/FabSim3/blob/radical/fabsim/base/env.py#L97) variables separately 😕



> Hi @cspgdds 😁 just got email for this MR, one comment (I just had a quick look), `lower()` method returns the lowercase string from the given string so, the `PJ_TYPE.lower()` variable should be compared with lowercase string
> 
> `env.PJ_TYPE.lower() == "QCG"` should be `env.PJ_TYPE.lower() == "qcg"`

Okay, I think I just resolved this issue whilst typing in a wobbly train. @mzrghorbani @cspgdds please retest as functionality will have been affected by this fix.

Good to hear from you again @arabnejad. Hope life is treating you well! :+1:

**Comment by djgroen** [comment]
Ok, this is ready for merging :)

**Pull request merged**